README.md

```diff
@@ -0,0 +1,158 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[In this project we have make a brand new modular system using ES6 syntax and a boundler like Webkit]**
+- **[You could find a dinamic website]**
+- **[CSS styles]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to clone this project you need:
+
+◦ Installation of Git
+◦ Installation of NPM
+◦ A working browser
+◦ Internet connection
+◦ A code editor like Visual Studio Code
+◦ Installation of Live Server plugging in Visual Studio Code
+
+### Setup
+
+To setup and install this project, follow the below steps:
+
+- Clone this project by the command:
+
+```
+$ git clone https://github.com/SaveryIV/Awesome-Books-Project.git
+```
+
+- Then switch to the project folder by the bellow query:
+
+```
+$ cd dir
+```
+
+### Install
+
+For setting up the linters you need:
+
+-Follow the steps mentionated in this article https://github.com/microverseinc/linters-config/tree/master/html-css
+
+### Usage
+
+To run the project:
+
+-Download the extension Live Server for Ritwick Dey.
+-Then close and open again VScode.
+-When you open the folder of the project in your local machine, in the lower left of your screen, you gonna see a item what says "Go Live".
+-Press that button and set the developers tools and select your toggle device toolbar.
+-And enjoy the project!.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Agustin Soria Meza**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[New styles with CSS]**
+- [ ] **[Use of APIs]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a⭐️
+
+If you like this project share it with others you think can help to learn something new
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Currently when your MIT license link is clicked it doesn't take the user to your license file because the link is broken, it shows the error seen in the image below. Please kindly make the necessary changes to correct this 👍 ✨ 
 ![al](https://github.com/SaveryIV/To-Do-List-ES6/assets/89055502/16a2b4ed-78cc-4aaf-9c84-c550a929395c)